### PR TITLE
first draft multinode script

### PR DESCRIPTION
**Pull request by kkraune**
@baldersheim or @geirst please merge - this is work in progress and essentially just a readme and an app package - but great to analyze document activation as reported in https://github.com/vespa-engine/vespa/issues/18194 - please merge and then we can look  into behavior, I don't understand the partial results with 2 nodes down

### CODE REVIEW

File: operations/multinode/src/main/application/schemas/music.sd [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```
Comment (geirst) [inline operations/multinode/src/main/application/schemas/music.sd:1]: Do we need to duplicate the schema file or can we use the schema file associated with the feed files: @../../album-recommendation-selfhosted/?

**Review by geirst — APPROVED** [review]
👍

File: operations/multinode/src/main/application/schemas/music.sd (hunk 1 above)
Comment (kkraune) [reply operations/multinode/src/main/application/schemas/music.sd:1]: good point, will symlink instead

Comment (kkraune) [reply operations/multinode/src/main/application/schemas/music.sd:1]: otoh, it is not duplicated but rather simplified (less rank profiles), so to have less deploy warnings etc will keep the schema

**Pull request merged**